pkg/api/goprobe/server/flows.go

```diff
@@ -0,0 +1,31 @@
+package server
+
+import (
+	"net/http"
+
+	gpapi "github.com/els0r/goProbe/pkg/api/goprobe"
+	"github.com/gin-gonic/gin"
+)
+
+func (server *Server) getFlows(c *gin.Context) {
+	iface := c.Param(ifaceKey)
+
+	resp := &gpapi.FlowsResponse{}
+	resp.StatusCode = http.StatusOK
+
+	// TODO: make sure that the all case is only hit when no iface is specified
+	// --> ?ifaces=eth0,eth1
+	if iface != "" {
+		resp.Flows = server.captureManager.ActiveFlows(iface)
+	} else {
+		// otherwise, fetch all
+		resp.Flows = server.captureManager.ActiveFlows()
+	}
+
+	if len(resp.Flows) == 0 {
```

Comment:
What if there's an error (e.g. HTTP 400) _and_ the number of flows is zero? Wouldn't you overwrite the error status code with `No Content`?